Cubical/Structures/Pointed.agda

```diff
@@ -24,6 +24,10 @@ pointed-iso : StrIso pointed-structure ℓ
 pointed-iso A B f = equivFun f (pt A) ≡ pt B
 
 pointed-is-SNS : SNS {ℓ} pointed-structure pointed-iso
-pointed-is-SNS {A = A} {B = B} f =
-  transportEquiv ( (λ j → transportRefl (equivFun f (pt A)) (~ j) ≡ pt B)
-                 ∙ sym (PathP≡Path (λ i → ua f i) (pt A) (pt B)))
+pointed-is-SNS f = invEquiv (ua-ungluePath-Equiv f)
```

Comment:
Simplified this slightly using glue types, hope that's okay.

In fact, I suspect that the proof of SIP could similarly be simplified (in the sense of removing `transport`'s) using glue/unglue. I would be happy to give it a try if it doesn't conflict with plans @mzeuner or @mortberg have.

Reply:
That would be cool! I don't think @mzeuner or I are planning to do anything about the SIP proofs at the moment. Please do it in a new PR if you succeed.